package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xkcd-password",
-  "version": "1.2.0",
+  "version": "2.0.0",
```

Comment:
please revert this line; i prefer to bump versions with `npm version major` on the CLI for the auto git tagging and etc.